src/Umbraco.Infrastructure/Migrations/Upgrade/V_13_0_0/AddGuidsToUsers.cs

```diff
@@ -0,0 +1,262 @@
+﻿using NPoco;
+using Umbraco.Cms.Core;
+using Umbraco.Cms.Infrastructure.Persistence.DatabaseAnnotations;
+using Umbraco.Cms.Infrastructure.Persistence.DatabaseModelDefinitions;
+using Umbraco.Cms.Infrastructure.Persistence.Dtos;
+using Umbraco.Cms.Infrastructure.Scoping;
+using Umbraco.Extensions;
+
+namespace Umbraco.Cms.Infrastructure.Migrations.Upgrade.V_13_0_0;
+
+/// <summary>
+/// This is an unscoped migration to support migrating sqlite, since it doesn't support adding columns.
+/// See <see cref="AddGuidsToUserGroups"/> for more information.
+/// </summary>
+public class AddGuidsToUsers : UnscopedMigrationBase
+{
+    private const string NewColumnName = "key";
+    private readonly IScopeProvider _scopeProvider;
+
+    public AddGuidsToUsers(IMigrationContext context, IScopeProvider scopeProvider)
+        : base(context)
+    {
+        _scopeProvider = scopeProvider;
+    }
+
+    protected override void Migrate()
+    {
+        using IScope scope = _scopeProvider.CreateScope();
+        using IDisposable notificationSuppression = scope.Notifications.Suppress();
+        ScopeDatabase(scope);
+
+        if (DatabaseType != DatabaseType.SQLite)
+        {
+            MigrateSqlServer();
+            scope.Complete();
+            return;
+        }
+
+        MigrateSqlite();
+        scope.Complete();
+    }
+
+    private void MigrateSqlServer()
+    {
+        var columns = SqlSyntax.GetColumnsInSchema(Context.Database).ToList();
```

Comment:
We should check if the column exists and return if it does. Just to make sure this migration is safe to call multiple times